revolv/project/stats.py

```diff
@@ -0,0 +1,124 @@
+"""
+Module for encapsulating functions related to project statistics (aka
+computing the effects of projects based on their stated kilowatt output).
+"""
+from django.db.models import Sum
+
+
+class KilowattStatsAggregatorException(Exception):
+    pass
+
+
+class KilowattStatsAggregator(object):
+    """
```

Comment:
Are these kilowatt hours or just kilowatts? And maybe link to a page that defines what specifically they are in regards to solar. Even after a year, still not fully sure of the distinction =P